### ISSUE TITLE
When Organizations are mapped to an application, SAML buttons should render accordingdly

### ISSUE THREAD

**Issue by dogeared** [stormpath/idsite-src#226]
![image](https://cloud.githubusercontent.com/assets/364562/16311153/6a9dd55a-393d-11e6-87fc-8d09a9f12979.png)

In the above scenario, if I go to: blue.wizbang.com, I should *only* see the `SAML Okta` button.

If I go to: green.wizbang.com, I should *only* see the `SAML Ping` button.

NOTE: This assumes that there are Stormpath cloud directories mapped to the organizations as well. IF there is only one provider, then login should put you directly into the flow of that provider - SAML in this case.

**Comment by lhazlewood** [comment]
Actually no intermediate screen should render - redirect to the SAML provider should occur automatically since there is only 1 directory.

**Comment by dogeared** [comment]
That's what the NOTE is for. ;) I will update the image. 

I think what you are speaking to @lhazlewood should be a separate issue as it doesn't just affect SAML. And, perhaps requires some further discussion.

For instance, if the *only* mapped Directory to an application is a Facebook Directory and there wasn't an intermediate screen with a Facebook button to click on, it might be a jarring experience for users to be dumped into a Facebook login flow.

At any rate, if that is the behavior we want, it's applicable outside of Organizations and SAML.

**Comment by lhazlewood** [comment]
yeah, we definitely need to talk about the desired approach depending on the provider type.  I think OAuth flows might make sense to have the intermediate page first, but SAML SSO flows usually redirect.  Let's discuss in the sdk strat meeting.
